tests/phpunit/api/v4/Entity/ConformanceTest.php

```diff
@@ -71,13 +71,9 @@ public function setUp() {
    */
   public function getEntitiesHitech() {
     // Ensure all components are enabled so their entities show up
-    \CRM_Core_BAO_ConfigSetting::enableComponent('CiviEvent');
```

Comment:
In setUp() here in this file, you _could_ add `(new \CiviCaseTestCase())->setUp();`. It feels icky and creates a different problem, but then with that and the $result change above most of it passes and it seems like it might just be a matter of working around the new problem.